crates/tauri-macros/Cargo.toml

```diff
@@ -22,11 +22,12 @@ syn = { version = "2", features = ["full"] }
 heck = "0.5"
 tauri-codegen = { version = "2.1.0", default-features = false, path = "../tauri-codegen" }
 tauri-utils = { version = "2.3.0", path = "../tauri-utils" }
+tauri-cli = { version = "2.4.0", path = "../tauri-cli" }
 
 [features]
 custom-protocol = []
 compression = ["tauri-codegen/compression"]
 isolation = ["tauri-codegen/isolation"]
-config-json5 = ["tauri-codegen/config-json5", "tauri-utils/config-json5"]
-config-toml = ["tauri-codegen/config-toml", "tauri-utils/config-toml"]
+config-json5 = ["tauri-codegen/config-json5", "tauri-cli/config-json5", "tauri-utils/config-json5"]
+config-toml = ["tauri-codegen/config-toml", "tauri-cli/config-toml", "tauri-utils/config-toml"]
```

Comment:
these need to be reverted, tauri-macros doesn't interact with tauri-cli at all.

tauri-cli is just a binary on its own.